app/views/comments/_comment.html.erb

```diff
@@ -10,12 +10,14 @@
       <p><%= comment.body %></p>
     </div>
     <div>
-      <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted" do %>
-        <i class="fas fa-edit fa-fw"></i>
-      <% end %>
+      <% if current_user == comment.author %>
+        <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted" do %>
+          <i class="fas fa-edit fa-fw"></i>
+        <% end %>
```

Comment:
Perfect use of wrapping editing function in the `current_user` if statement to keep them from changes comments that are not their own!